shared/network/2.x.md

```diff
@@ -367,8 +367,20 @@ Additionally, you should whitelist the following domains for the relevant ports
 * `*.{{ $names.dashboard_domain }}`
 * `*.docker.com`
 * `*.docker.io`
-* `*.resinio.pool.ntp.org`
 
+NTP / UDP packets (port 123) are exchanged with:
+
+* `0.resinio.pool.ntp.org`
+* `1.resinio.pool.ntp.org`
+* `2.resinio.pool.ntp.org`
+* `3.resinio.pool.ntp.org`
+
+[Google's Public DNS](https://developers.google.com/speed/public-dns) server at 8.8.8.8 is used by
+default and in addition to DNS servers obtained via DHCP (balenaOS may issue queries to multiple
+DNS servers simultaneously, for the quickest response to be used). If additional DNS servers are
+configured via DHCP or other means, it is OK for the local network to block requests to `8.8.8.8`.
+To avoid any requests being made to `8.8.8.8` by balenaOS, modify the [dnsServer
+setting]((https://github.com/balena-os/meta-balena#dnsservers)) in `config.json`.
```

Comment:
@majorz, @ZubairLK, is this correct? It's my understanding / recollection, but I couldn't find any documentation for it... :-)  It addresses @xginn8's issue #1122

Reply:
@pdcastro that is correct. More detail here about the current dnsServer config.json behaviour
https://github.com/balena-os/meta-balena#dnsservers